build/lib/release.sh

```diff
@@ -341,31 +365,31 @@ function kube::release::create_docker_images_for_server() {
     # provide `--pull` argument to `docker build` if `KUBE_BUILD_PULL_LATEST_IMAGES`
     # is set to y or Y; otherwise try to build the image without forcefully
     # pulling the latest base image.
-    local DOCKER_BUILD_OPTS=()
+    local docker_build_opts
+    docker_build_opts=
```

Comment:
is there a reason to not continue using an array?
or possibly we should rename this to `docker_maybe_pull`?

Reply:
Had it not been for shellcheck failing on the current code, I would not have changed it. And since the use is either empty array or an array with 1 item, I think there is little need to use an array at all. Will there be a lot of other options in the future? Perhaps, and then it might be a good idea to do a shellcheck-friendly array solution, but as the code stands today I reckon keeping it simple is a better choice.

Reply:
I'm not sure, were there ever other options in the history?
simpler SGTM otherwise